.github/workflows/release-to-edge.yml

```diff
@@ -0,0 +1,121 @@
+name: Release
```

Comment:
I think this file needs to be removed - we don't typically release to `edge`. The call for testing parts can be taken from this file and added to the candidate release flows